types/2020-08-27/QuotePhases.d.ts

```diff
@@ -0,0 +1,226 @@
+// File generated from our OpenAPI spec
+
+declare module 'stripe' {
+  namespace Stripe {
+    /**
+     * The QuotePhase object.
+     */
+    interface QuotePhase {
+      /**
+       * Unique identifier for the object.
+       */
+      id: string;
+
+      /**
+       * String representing the object's type. Objects of the same type share the same value.
+       */
+      object: 'quote_phase';
+
+      /**
+       * Total before any discounts or taxes are applied.
+       */
+      amount_subtotal: number;
+
+      /**
+       * Total after discounts and taxes are applied.
+       */
+      amount_total: number;
+
+      /**
+       * If set to `new`, the billing_cycle_anchor of the subscription is set to the start of the phase when entering the phase. If unset, then the billing cycle anchor is automatically modified as needed when entering the phase. For more information, see the billing cycle [documentation](https://stripe.com/docs/billing/subscriptions/billing-cycle).
+       */
+      billing_cycle_anchor: 'reset' | null;
```

Comment:
How can this be `new`?

Reply:
Going to follow up in a second PR. Great catch!